danger.c

```diff
@@ -0,0 +1,7 @@
+# crosstest
+I like banned API methods like strncpy()
+
+Not to mention deprecated crypto algorithms like MD5
```

Comment:
[Weak/Broken Hash Algorithm](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS126858.md#DS126858)
A weak or broken hash algorithm was detected.

*Fix Guidance:* Consider switching to use SHA-256 or SHA-512 instead.


Suggestions:
[1] Not to mention deprecated crypto algorithms like SHA256
[2] Not to mention deprecated crypto algorithms like SHA512